usr/sbin/pkcsep11_migrate/pkcsep11_migrate.c

```diff
@@ -37,16 +37,25 @@ CK_LONG adapter = -1;
 CK_LONG domain = -1;
 CK_OBJECT_HANDLE key_store[4096];
 
-int (*_m_get_ep11_info) (CK_VOID_PTR pinfo, CK_ULONG_PTR, unsigned int,
-                         unsigned int, uint64_t);
-long (*_ep11a_cmdblock) (unsigned char *, size_t, unsigned int,
-                         const struct ep11_admresp *, const unsigned char *,
-                         const unsigned char *, size_t);
-unsigned long int (*_m_admin) (unsigned char *, size_t *, unsigned char *,
-                               size_t *, const unsigned char *, size_t,
-                               const unsigned char *, size_t, uint64_t);
-long (*_ep11a_internal_rv) (const unsigned char *, size_t,
-                            struct ep11_admresp *, CK_RV *);
+typedef int (*m_get_ep11_info_t) (CK_VOID_PTR, CK_ULONG_PTR,
+                                  unsigned int, unsigned int, uint64_t);
+typedef unsigned long int (*m_admin_t) (unsigned char *, size_t *,
+                                        unsigned char *,
+                                        size_t *, const unsigned char *,
+                                        size_t, const unsigned char *,
+                                        size_t, uint64_t);
+typedef long (*ep11a_cmdblock_t) (unsigned char *, size_t, unsigned int,
+                                  const struct ep11_admresp *,
+                                  const unsigned char *,
+                                  const unsigned char *, size_t);
+typedef long (*ep11a_internal_rv_t) (const unsigned char *, size_t,
+                                     struct ep11_admresp *, CK_RV *);
+
+m_get_ep11_info_t _m_get_ep11_info;
```

Comment:
static ?

Reply:
The tools under sbin/ have a lot of non-static global vars. Im not going to fix it whith this PR.